packages/babel-runtime/scripts/build-dist.js

```diff
@@ -88,9 +115,10 @@ each(helpers.list, function (helperName) {
 
   // compat
   var helperAlias = _.kebabCase(helperName);
-  writeFile("helpers/_" + helperAlias + ".js", buildHelper(helperName));
-  writeFile("helpers/" + helperAlias + ".js", buildHelper(helperName));
+  var content = "module.exports = require(\"./" + helperName + ".js\");";
+  writeFile("helpers/_" + helperAlias + ".js", content);
+  if (helperAlias !== helperName) writeFile("helpers/" + helperAlias + ".js", content);
```

Comment:
The helpers are all here: https://github.com/babel/babel/blob/master/packages/babel-helpers/src/helpers.js